manim/config/config_utils.py

```diff
@@ -69,9 +69,15 @@ def _parse_file_writer_config(config_parser, args):
             default.getboolean(boolean_opt) if attr is None else attr
         )
     # for str_opt in ['media_dir', 'video_dir', 'tex_dir', 'text_dir']:
-    for str_opt in ["media_dir", "log_dir"]:
+    for str_opt in ["media_dir"]:
         attr = getattr(args, str_opt)
         fw_config[str_opt] = os.path.relpath(default[str_opt]) if attr is None else attr
+    attr = getattr(args, "log_dir")
+    fw_config["log_dir"] = (
+        os.path.join(fw_config["media_dir"], default["log_dir"])
+        if attr is None
+        else attr
+    )
```

Comment:
Question: I don't understand why this is put here and the default.cfg was changed from `%(media_dir)s/logs` to just `logs`. These two seem to have the same result.

Reply:
IT does not work otherwise for a strange reason, and I think I just discovered a new issue. 
When passing `--media_dir` as flag, `log_dir` (and I think tex(t)_dir as well), their paths are not updated with the media_dir passed as flag. 

To confirm tho

Reply:
I noticed this as well, although for images and videos, this is the reason why I had to set their directories separately in https://github.com/ManimCommunity/manim/pull/402/files#diff-0d9eb0ee2926ea2e8371ae8f9842a28dR155-R157

Reply:
Oh I see yeah, this is a bug of the stupid config system. Let's leave this change in since the config system is being totally revamped soon either way.